src/sources/pulsar.rs

```diff
@@ -0,0 +1,612 @@
+use crate::codecs::{Decoder, DecodingConfig};
```

Comment:
Would it be possible to add some module level documentation. Include relevant links to the pulsar documentation and the pulsar crate.